### ISSUE TITLE
Not able to add volume through cli command by using --volume parameter on docker-compose v2

### ISSUE THREAD

**Issue by sliwinski-milosz** [docker/compose#8595]
I am trying to add the volume to the docker-compose run command by passing `--volume` flag as described [here](https://docs.docker.com/compose/reference/run/) but I get:
```
unknown flag: --volume
```

Please note, that I have tried the short version of the parameter: `-v` and it works fine.

```
$ docker-compose --version
Docker Compose version v2.0.0-rc.2
$ docker --version
Docker version 20.10.8, build 3967b7d
```

I have tested on the docker-compose v1 and the `--volume` parameter (long one) works fine:
```
$ docker-compose --version
docker-compose version 1.29.2, build 5becea4c
```

EDIT: I realised it has been renamed from `--volume` to `--volumes`. Is that expected?
https://github.com/docker/compose/blob/v2.0.0-rc.3/cmd/compose/run.go#L48

**Comment by ndeloof** [comment]
unexpected, this is a mistake we made replicating the legacy `docker-compose run` command.

**Comment by mat007** [comment]
Fixed in #8596
